main/EDTypesMod.F90

```diff
@@ -146,6 +146,7 @@ module EDTypesMod
   integer,  parameter, public :: lg_sf                = 6          ! array index of live grass pool for spitfire
 
   real(r8), parameter, public :: fire_threshold       = 50.0_r8    ! threshold for fires that spread or go out. KWm-2 (Pyne 1986)
+  real(r8), parameter, public :: crown_fire_threshold = 200.0_r8   ! threshold for passive crown fire ignition. KWm-2 (Bessie & Johnson 1995)
```

Comment:
Just noticed that this parameter appears here, as well as in SFMainMod line 917.

Reply:
@slevisconsulting yep. that was my debugging frustration. corrected this by bringing variable into SFMain from EDTypes near top and removing line 917 in SFMain